docs/payment_flow.txt

```diff
@@ -0,0 +1,30 @@
+# Generated with https://sequencediagram.org/
+
+title Venepaikka payment flow
+
+note over Admin: Confirm order
+Admin->Back-end: confirmOrderMuation
+Back-end->User: Send mail to Customer UI with order number\nhttps://venepaikat.hel.fi/{LANG}/payments?order_number={str}
```

Comment:
path will be `payment`

Reply:
for the sake of clarity, I would maybe say here `Send email with the following link to Customer UI` 🤔